pkg/operator/ceph/pool/controller.go

```diff
@@ -149,6 +149,7 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 		logger.Debugf("deleting pool %q", cephBlockPool.Name)
 		err := deletePool(r.context, cephBlockPool)
 		if err != nil {
+			logger.Errorf("could not delete pool %q. %v", cephBlockPool.Name, err)
```

Comment:
This shouldn't be necessary, is it a left-over?

Reply:
I didn't see any logging when a pool was failing to be deleted. Seems like a good thing to log.